### DISCUSSION TITLE
A status bar theme accent that changes with status

### DISCUSSION THREAD

**Discussion by lilazero** [vscode-neovim/vscode-neovim#2310] [Ideas]
Like it's done in VsCodeVim. I know there's an extension but it barely works and has been discontinued

**Comment by theol0403** [comment]
You can write an autocommand that modifies the vscode setting through the API.

**Comment by wrath-codes** [comment]
Not sure if it's useful @lilazero , but i made an extension to update the ui based on the current neovim mode [Nvim UI+](https://marketplace.visualstudio.com/items?itemName=wrathcodes.nvim-ui-plus)
